gemrb/core/Projectile.cpp

```diff
@@ -1697,7 +1697,9 @@ void Projectile::DrawTravel(const Region &screen)
 	// set up the tint for the rest of the blits, but don't overwrite the saved one
 	Color tint2 = tint;
 	ieDword flags = flag;
-	if (game) game->ApplyGlobalTint(tint2, flags);
+	if (TFlags&PTF_TINT) {
+		if (game) game->ApplyGlobalTint(tint2, flags);
+	}
```

Comment:
Not really sure about this, but self-illuminating projectiles are otherwise far too dark at night.

Reply:
What were your test cases?

Reply:
I basically tested two things: Various distance spells (magic missile, lightning) and flame arrows.